README.md

```diff
@@ -94,22 +95,75 @@ stanza is needed and should specify the number of GPUs available.
 snakemake --profile my-profile
 ```
 
-### Relevant output files
+### Output files
+
+.fastq files are archived as compressed file pair as .spring: `Archive/{project}/{sample}_{flowcell}_{lane}_{barcode}_{type}.spring`
+
+The MultiQC html report can be found here: `Results/MultiQC_TN.html`
+
+All results (as described in table below) are located in: `Results/{project}/{sample}/`
 
 | File | Description |
 |---|---|
-| `cnv_sv/cnvkit_diagram/{sample}_T.png` | chromosome diagram from cnvkit |
-| `cnv_sv/cnvkit_scatter/{sample}_T_{chromosome}.png` | scatter plot per chromosome from cnvkit |
-| `cnv_sv/cnvkit_vcf/{sample}_T.vcf` | `.vcf` output from cnvkit |
-| `cnv_sv/pindel/{sample}.vcf` | `.vcf` output from pindel |
-| `compression/crumble/{sample}_{type}.crumble.cram` | crumbled `.cram` file |
-| `compression/crumble/{sample}_{type}.crumble.cram.crai` | index for crumbled `.cram` file |
-| `compression/spring/{sample}_{flowcell}_{lane}_{barcode}_{type}.spring` | compressed `.fastq` file pair |
-| `tsv_files/{sample}_mutectcaller_tn.aml.tsv` | `.tsv` file for excel containing SNVs from mutect2 for AML |
-| `tsv_files/{sample}_mutectcaller_tn.all.tsv` | `.tsv` file for excel containing SNVs from mutect2 for ALL |
-| `tsv_files/{sample}_manta_tn.aml.tsv` | `.tsv` file for excel containing SVs from manta for AML |
-| `tsv_files/{sample}_manta_tn.all.tsv` | `.tsv` file for excel containing SVs from manta for ALL |
-| `qc/multiqc/multiqc_DNA.html` | `.html` report from MultiQC |
+| `Cram/{sample}_{type}.crumble.cram` | crumbled `.cram` file |
+| `Cram/{sample}_{type}.crumble.cram.crai` | index for crumbled `.cram` file |
+| `SNV_indels/{sample}_T.vep.vcf.gz` | `.vcf` output from VEP for tumor_only |
```

Comment:
maybe something like:
`.vcf output for snv and small indels annotated with vep for tumor only

Reply:
done